src/_guides/bump-sh-tutorials/api-platform.md

```diff
@@ -13,17 +17,45 @@ The following assumes your local machine is configured with PHP and API Platform
 1. [Create and name](https://bump.sh/docs/new?utm_source=bump&utm_medium=content_hub&utm_campaign=getting_started) your first API documentation. Then, retrieve the name and token of this documentation from the _CI deployment_ settings page.
 
 2. Install the Bump.sh CLI with [npm](https://docs.npmjs.com/cli/v9/configuring-npm/install?v=true) as below, or use [alternative options](/help/continuous-integration/cli), with
-  ```bash
+  ```shell
   npm install -g bump-cli
   ```
 
 3. Launch your local server with [Docker](https://api-platform.com/docs/distribution/#using-the-api-platform-distribution-recommended) or [Symfony](https://api-platform.com/docs/distribution/#using-symfony-cli)
 
 4. Deploy your doc to Bump.sh with
-  ```bash
+  ```shell
   bump deploy https://localhost/docs.json \
     --doc my-documentation-name \
     --token my-documentation-token
   ```
 
 That's it! Enjoy the comfort of Bump.sh to browse through your API doc, and [customize it to your needs](/help/getting-started/quick-start#customization-options).
+
+## Enriching definition files
+
+By default, API Platform generates what can be considered a plain technical description of your API. Although it is robust and complete, it's not enough for people discovering your API without any upfront knowledge about it, access to the code, or ability to test it extensively. Such definition files require contextual information, and any appropriate adjustements.
+
+API Platform lets you override, enrich and filter the generated OpenAPI definition files, [using PHP](https://api-platform.com/docs/core/openapi/). 
+
+You can also accomplish this using OpenAPI's [Overlay specification](https://github.com/OAI/Overlay-Specification/blob/main/versions/1.0.0.md), which can be convenient in all use cases where you do not want to change your PHP code base (e.g. when involving Technical Writers or Product Managers in writing docs).
+
+The Bump.sh CLI supports Overlays. Technically, the `bump overlay` command will output a modified version of the `[DEFINITION_FILE]` (an OpenAPI or AsyncAPI document) by applying the operations described in the `[OVERLAY_FILE]` Overlay file to the original API document.
+
+1. Create your Overlay document (see example use cases in [Augmenting Generated OpenAPI Documents with Filters & Overlays](/guides/openapi/augmenting-generated-openapi))
+
+2. Apply your Overlay document to the original API document with the Bump.sh CLI
+```shell
+bump overlay docs.json overlay-file.json > api-overlayed-document.json
```

Comment:
I'd stick to YAML and use standard names similar to the guide we wrote: 

```suggestion
bump overlay openapi.yaml overlays.yaml > openapi.updated.yaml
```

Reply:
I opted for JSON and the initial `docs.json` file name as it's the file name and format that is automatically generated by API Platform.

I wanted to make API Platform user feel more "at home", WDYT?

Reply:
Good point! Yeah best go with whatever their defaults are, unless you'd like to help them have a nicer experience. 

Or you can update the guide to help them make more standard output:

<img width="660" alt="image" src="https://github.com/bump-sh/docs/assets/67381/4e7a9800-a172-4a0c-baf5-9cf1bcef7434">


Either way, the `overlays.yaml` name and `openapi.something.yaml` is up to us so could be whatever.